CONTRIBUTING.md

```diff
@@ -39,9 +39,20 @@ Guidelines for rosdep rules
 ---------------------------
 
  * Native packages are strongly preferred (They are required for packaging and have upgrade and conflict tracking.)
- * Python packages should go in the python.yaml
- * Homebrew into homebrew.yaml
+ * Please use the smallest notation possible and if something is default or not necessary to specify, don't.
+ * Python packages should go in the python.yaml.
```

Comment:
nitpick: The use of `.` is very inconsistent, you added it here, but on the next line (which you also updated) it doesn't have it. Use it or don't, but being consistent is ideal.